contracts/ThalesRoyale/ThalesRoyale.sol

```diff
@@ -82,26 +84,27 @@ contract ThalesRoyale is Initializable, ProxyOwned, PausableUpgradeable, ProxyRe
         bytes32 _oracleKey,
         IPriceFeed _priceFeed,
         address _rewardToken,
+        address _royaleVoucher,
         uint _rounds,
         uint _signUpPeriod,
         uint _roundChoosingLength,
         uint _roundLength,
         uint _buyInAmount,
-        uint _pauseBetweenSeasonsTime,
-        bool _nextSeasonStartsAutomatically
-    ) public initializer {
+        uint _pauseBetweenSeasonsTime    
+        ) 
+        public initializer {
         setOwner(_owner);
         initNonReentrant();
         oracleKey = _oracleKey;
         priceFeed = _priceFeed;
         rewardToken = IERC20Upgradeable(_rewardToken);
+        royaleVoucher = IThalesRoyaleVoucher(_royaleVoucher);
```

Comment:
afaik initializer method cant change as the proxy is already deployed.

Reply:
roayeVoucher should be set through a setter

Reply:
ah ok, will do